docs/build-from-scratch.md

```diff
@@ -32,6 +34,12 @@ sudo make all
 
 To push your image to a registry, use `make push`.
 
+### Keeping the Heptio registry tag
+
+If you don't wish to push your image to your own registry (not changing `$REGISTRY`), but simply want to build it locally with the Heptio tag, *you need to modify the deployment file that are using.* This is either `examples/azure/00-ark-deployment.yaml` if you're using Azure, or `examples/common/10-deployment.yaml` for all other cloud providers.
```

Comment:
> with the Heptio tag

This isn't the right terminology. The gist here is that if you're building locally and your tag is "latest", you have to change the image pull policy or the image you're deploying. Maybe restructure the paragraph around the concept of building & testing locally?